phone_verify/models.py

```diff
@@ -34,6 +34,7 @@ class SMSVerification(TimeStampedUUIDModel):
     security_code = models.CharField(_("Security Code"), max_length=120)
     phone_number = PhoneNumberField(_("Phone Number"))
     session_code = models.CharField(_("Device Session Code"), max_length=500)
+    is_verified = models.BooleanField(_("Security Code Verified"), default=True)
```

Comment:
The default should be `False`. We don't want to generate security codes that are already verified.

Reply:
Yes, for newly generated security codes, the default value for `is_verified` is given to be `False`. [See this](https://github.com/CuriousLearner/django-phone-verify/pull/16/files#diff-9f924f5888b070e5444e4db988177019R76-R78).

But here, in the models I have given default for `is_verified` as `True` so that, existing projects which are using this package will get `is_verifed` as `True` corresponding to the entries already present in their database.

Reply:
But why would they need all entries marked as `is_verified`?

It should be `is_verified` as `False` in all the cases.

Reply:
Okay, I just thought that their existing entries of security codes would already be verified so they all should be having `is_verified` as `True`. But I got your point, we never know if they are verified or not so we should set `is_verified` to `False` by default.